common/lib/xmodule/xmodule/combined_open_ended_module.py

```diff
@@ -515,3 +525,22 @@ def non_editable_metadata_fields(self):
     # Proxy to CombinedOpenEndedModule so that external callers don't have to know if they're working
     # with a module or a descriptor
     child_module = module_attr('child_module')
+
+    def validate(self):
+        """
+        Validates the state of this instance. This is the override of the general XBlock method,
+        and it will also ask its superclass to validate.
+        """
+        validation = super(CombinedOpenEndedDescriptor, self).validate()
+        validation = StudioValidation.copy(validation)
+
+        _ = self.runtime.service(self, "i18n").ugettext  # pylint: disable=redefined-outer-name
```

Comment:
And then call `i18nService.ugettext()` below. It seems nicer than disabling a pylint violation.